README.md

```diff
@@ -87,7 +87,7 @@ bal run
 
 The `OpenAI Images` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-openai.images/tree/main/examples/), covering the following use cases:
 
-1. [Personalized visual content generator](https://github.com/ballerina-platform/module-ballerinax-openai.images/tree/main/example/Personalized-visual-content-generator) - Create personalized visual content for targeted email marketing by generating and editing images to include specific products.
+1. [Personalized visual content generator](https://github.com/ballerina-platform/module-ballerinax-openai.images/tree/main/examples/Personalized-visual-content-generator) - Create personalized visual content for targeted email marketing by generating and editing images to include specific products.
```

Comment:
directory names should start with lowercase. Please update the example directories

```suggestion
1. [Personalized visual content generator](https://github.com/ballerina-platform/module-ballerinax-openai.images/tree/main/examples/personalized-visual-content-generator) - Create personalized visual content for targeted email marketing by generating and editing images to include specific products.
```

Reply:
Changes Applied